lomap/graphgen.py

```diff
@@ -901,15 +963,23 @@ def max_dist_mol(mol):
             for each_node in self.resultGraph:
                 
                 id_mol = self.resultGraph.node[each_node]['ID']
-                mol = AllChem.RemoveHs(self.dbase[id_mol].getMolecule())
+                #sliu 07/17 skip remove Hs by rdkit if Hs cannot be removed
```

Comment:
When rdkit try to remove hydrogens, it will kekulize the molecule object, while the rdkit will have errors to kekulize molecules (for most cases I saw is the heterocycle). If the RemoveHs is not applied, the kekulized process will be skip so the molecule object will be passed for later layout but in this case, it will have Hydrogens but it looks fine I think.